src/css/SocialFollow.css

```diff
@@ -63,13 +64,31 @@
         background-color: #333333;
         transition: 0.5s;
         transform: translate(0,0px);
+        transform: scale(0.8);
+    }
+
+    .Social ul li:nth-child(2) img{
+        transform: scale(0.8);
+        transform: translateY(-5%);
+    }
+
+    .Social ul li:nth-child(3) img {
+        transform: scale(0.8);
+        transform: translateY(10%);
     }
     
     .Social ul li:last-child a{
         position: relative;
         display: inline-block;
         width: 120px;
         border-radius: 2%;
+        line-height: 130%;
     }
+}
 
+/* MAKE ICONS DISAPPEAR */
+@media screen and (max-width:1300px) {  
```

Comment:
UPO CHANGE THIS should be 600 px it needs to disappear for mobile not desktop

Reply:
Yep it's changed